testsuite/testcases/src/test/cases/over-2gb-input/.gitignore

```diff
@@ -0,0 +1 @@
+Over2GbInput.java
```

Comment:
Also, I think you didn't intent to submit `Over2GbInput-0.input`

Reply:
I did mean to, actually.  This is the input file that is over 2GB.  It would be better to generate it at test time, though that slow the test down quite a bit.  I'll look into generating at test time.

Reply:
OK, I think such a large file in git is not a good idea. I think a quick workaround is to zip that file, and open the zipped version in the test.